pkg/test/framework/components/echo/config.go

```diff
@@ -72,6 +72,19 @@ type Config struct {
 	// ReadinessTimeout specifies the timeout that we wait the application to
 	// become ready.
 	ReadinessTimeout time.Duration
+
+	// Workloads contains the list of workloads config belonging to this echo
+	// service instance.
+	Workloads []WorkloadConfig
```

Comment:
I still feel this is the wrong term. It's not defining the config for a "workload", it's more like defining a config for a "subset". A workload is a single backend, but we're defining n backends that all have the same config. In k8s this is a "deployment".

Reply:
But we are not distinguishing the differencs of each individual replica instane vs template very much. For example, the workload selector in various API. The upcoming `WorkloadEntry` actually has very good mapping to k8s deployments.

I can change it to deployment if this macthes the testing framework terms better, but that seems a bit deviate from api terminology as far as I understand..

Reply:
I'm not sure I follow your point here. In what way is this not a "subset"?

Reply:
oh, i missed your point. subset is fine to me as well. done.